Goobi/src/de/sub/goobi/metadaten/Metadaten.java

```diff
@@ -3098,15 +3099,59 @@ boolean canCreate(MetadataGroupType type) {
      *             multi-select input
      */
     public List<RenderableMetadataGroup> getMyGroups() throws ConfigurationException {
-        List<MetadataGroup> records;
-        if (myDocStruct == null || (records = myDocStruct.getAllMetadataGroups()) == null) {
+        if (myDocStruct == null) {
             return Collections.emptyList();
         }
+
+        List<MetadataGroup> records = myDocStruct.getAllMetadataGroups();
+        if (records == null) {
+            records = Collections.emptyList();
+        }
+        List<MetadataGroupType> defaultGroupTypes = myDocStruct.getType().getAllDefaultDisplayMetadataGroups();
+        if (records.isEmpty() && defaultGroupTypes.isEmpty()) {
```

Comment:
shouldn't defaultGroupTypes be checked for null?

Reply:
No.